### ISSUE TITLE
Quick Start Guide: donejs build cordova command error

### ISSUE THREAD

**Issue by BigAB** [donejs/donejs#174]
In the **Desktop and mobile apps** section:

I get an error when running `donejs build cordova`.

```
$ donejs build cordova

...snip...

Error: Command `npm run build cordova run build cordova` did not complete successfully
    at ChildProcess.<anonymous> (/Users/adamlbarrett/.nvm/v0.10.36/lib/node_modules/donejs/lib/utils.js:61:23)
    at ChildProcess.emit (events.js:98:17)
    at Process.ChildProcess._handle.onexit (child_process.js:820:12)
```

It appears `donejs build cordova` doubles up the `run build cordova` command or somthing.
```
npm run build cordova run build cordova
```
I was able to continue on by typing `npm run build cordova` and continue working, but others might not read the error message

**Comment by matthewp** [comment]
Thanks, that's a bug then.

**Comment by daffl** [comment]
I think the reporting is just an error when concatenating the string at https://github.com/donejs/donejs/blob/master/lib/utils.js#L62. It does run the correct command but I've seen this error before. Strangely everything still works and other times I ran it worked fine.

**Comment by daffl** [comment]
Ok, the error reporting should be fixed in DoneJS v0.5.0-pre.14.
From what I could find out the process only exits unsuccessfully when the `build/` folder doesn't exist. Still investigating.

**Comment by daffl** [comment]
Fixed via 756fa358a6fd92eb00f9626a371c1a3415f978f8 in v0.5.0-pre.15.
